articles/anomaly-detection/passwords-checking.md

```diff
@@ -0,0 +1,229 @@
+Bulk Passwords Checking
```

Comment:
Throughout -- please use the terms Bulk Password Checking and Breached-Password Detection. Note the dash in the second one, but no dash in the first.